bundle/manifests/cluster-group-upgrades-operator.clusterserviceversion.yaml

```diff
@@ -170,7 +170,7 @@ spec:
                 - --leader-elect
                 command:
                 - /manager
-                image: quay.io/openshift-kni/cluster-group-upgrades-operator:v0.0.3
+                image: quay.io/alosadag/cluster-group-upgrades-operator:latest
```

Comment:
Looks like you forgot to change it back to openshift-kni, same as below.
Also I think the image tag should still be v0.0.3, in the Makefile,  we have 
VERSION ?= 0.0.3
BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)

Reply:
Fixed, thank you @Missxiaoguo